framework/doc/content/source/neml2/userobjects/NEML2FEInterpolation.md

```diff
@@ -0,0 +1,58 @@
+# NEML2FEInterpolation
```

Comment:
do you envision this interpolation to do/gather (I ll know when I look at the code) for only ever elementsxqps

or will it do side x qps, lower D x qps, nodes, physical points ...

Reply:
That's a good point. They currently only do element loops, not including sides. If later there's a need, we need to add `NEML2SideAssembly` and `NEML2SideFEInterpolation`.